certificate-policy.md

```diff
@@ -108,7 +108,7 @@ The primary goal of these Requirements is to enable efficient and secure electro
 
 
 ### 1.4.2 Prohibited Certificate Uses
-All Person identity certificates including but not limited to Person certificates used for digital signature, S/MIME, person authentication, and encryption. Trusted Role certificates MAY be issued from either the Root or Subordinate CA.
+All Person identity certificates including but not limited to Person certificates used for digital signature, S/MIME, person authentication, and encryption unless issued for specific infrastructure purposes as outlined in section 6.1.7.
```

Comment:
Additional question on this edit; 

- Edit references 6.1.7 - which is specific to the Root CA
- Doesn't this apply to subordinate and issuing CAs too? Why limit to 6.1.7?